src/solutions/how-tos/confluent-cloud/iot-ingest-control.md

```diff
@@ -0,0 +1,455 @@
+---
+icon: aky-zilla-plus
+description: Setup a public to your Confluent Cloud private cluster from anywhere on the internet.
```

Comment:
Setup a public ?? to your Confluent Cloud...

This probably just needs to be reworded for IoT solution.